From patchwork Mon Jan 18 15:14:43 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
Subject: cpupower: Fix build error in cpufreq-info
From: "Shreyas B. Prabhu" <shreyas@linux.vnet.ibm.com>
X-Patchwork-Id: 8055301
Message-Id: <1453130083-16055-1-git-send-email-shreyas@linux.vnet.ibm.com>
To: trenn@suse.com
Cc: linux-pm@vger.kernel.org, linux-kernel@vger.kernel.org,
	rafael.j.wysocki@intel.com, jtanenba@redhat.com,
	"Shreyas B. Prabhu" <shreyas@linux.vnet.ibm.com>
Date: Mon, 18 Jan 2016 20:44:43 +0530

Fix the following build error by including limits.h -

utils/cpufreq-info.c: In function ‘get_latency’:
utils/cpufreq-info.c:437:29: error: ‘UINT_MAX’ undeclared (first use in
this function)
  if (!latency || latency == UINT_MAX) {
                             ^
Signed-off-by: Shreyas B. Prabhu <shreyas@linux.vnet.ibm.com>
Fixes: e98f033f94f3 (cpupower: fix how "cpupower frequency-info" interprets latency)

---
tools/power/cpupower/utils/cpufreq-info.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/power/cpupower/utils/cpufreq-info.c b/tools/power/cpupower/utils/cpufreq-info.c
index 8f3f5bb9c74e..590d12a25f6e 100644
--- a/tools/power/cpupower/utils/cpufreq-info.c
+++ b/tools/power/cpupower/utils/cpufreq-info.c
@@ -10,6 +10,7 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
+#include <limits.h>
 
 #include <getopt.h>
 
